src/azul/openapi.py

```diff
@@ -2,23 +2,65 @@
 from typing import List
 
 from azul.deployment import aws
-from azul.types import JSON
+from azul.types import (
+    JSON,
+    MutableJSON,
+)
 
 
-def openapi_spec(description: JSON):
+def openapi_spec(path: str, methods: List[str], path_spec: MutableJSON, method_spec: MutableJSON):
```

Comment:
Shouldn't `path_spec` be optional?

Reply:
In fact, I think both need to be optional. We should force users to pass these as keyword arguments. We should programmatically enforce that at least one of the two is passed.

Reply:
I added this and tests.

Reply:
But actually I have to remove this behaviour for now. If we prevent both from being None, then any existing endpoints without documentation will cause this assertion to fail. Once all endpoints are documented, we can consider adding this back in.